multihost_test/Bugzillas/test_subid_ranges_configurable.py

```diff
@@ -0,0 +1,294 @@
+""" Automation of Shadow-utils bugs """
+
+import pytest
+import subprocess
+import time
+import os
+from sssd.testlib.common.utils import SSHClient
+
+
+def exceute_cmd(multihost, command):
+    """ execute cmd in client machine """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+class TestSubid(object):
+    """
+    This is for Shadow bugs automation
+    """
+    def test_subid_nochange_nsswitch(self, multihost,
+                                     create_backup,
+                                     create_localuser):
+        """
+        :Title: support pluggable data sources for
+        subid ranges configurable via /etc/nsswitch.conf
+        :id: 6ff93666-00c0-11ec-9be4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1859252
+        :steps:
+            1. Define user range in /etc/subuid
+            2. Define group range in /etc/subgid
+            3. Create a process for user
+            4. Check newuidmap command is working
+            with no change in /etc/nsswitch.conf
+            5. Check newgidmap command is working
+            with no change in /etc/nsswitch.conf
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+            4. Should succeed
+            5. Should succeed
+        """
+        multihost.client[0].run_command("echo 'local_anuj:165536:65536'"
```

Comment:
I'm not sure `execute cmd` helper makes a lot of sense, but perhaps it could be used here?

Reply:
fixed